### PR TITLE
adding 2024 folder, removing redundant cfp page and updating organizers

### PR DESCRIPTION
**Pull request by pmusau17**
adding 2024 folder, removing redundant cfp page and updating organizers

### CODE REVIEW

File: neurips2024/index.md
Comment (NorthStar) [inline neurips2024/index.md:1]: Please `git rm index.md` before committing again.

File: neurips2024/schedule.md
Comment (NorthStar) [inline neurips2024/schedule.md:1]: Please `git rm schedule.md` before committing again.

**Review by NorthStar — CHANGES_REQUESTED** [review]
Hi patrick, thanks for the PR. Please remove the neurips2024 folder, which seems redundant. We keep the most updated ("current") year's programme in the uppper-level directory (i.e. just `index.md` and just `call_for_papers.md`). So please make the CFP changes there.

**Comment by pmusau17** [comment]
Thanks Mimee.  Done.

**Comment by pmusau17** [comment]
Re-opening.

**Review by NorthStar — APPROVED** [review]

**Pull request merged**